tower/src/util/rng.rs

```diff
@@ -116,6 +116,7 @@ where
 ///
 /// ref: This was borrowed and modified from the following Rand implementation
 /// https://github.com/rust-random/rand/blob/b73640705d6714509f8ceccc49e8df996fa19f51/src/seq/index.rs#L375-L411
+#[cfg(any(feature = "full", not(any(feature = "hedge", feature = "retry", feature = "spawn-ready", feature = "util"))))]
```

Comment:
It looks like this method is only used in `Balance`, so it seems like the stanza should just be:

```suggestion
#[cfg(feature = "balance")]
```

Reply:
@tobz, thank you for pointing that out but when I checked the build logs, the warning for `sample_floyd2` appearded on `hedge, retry, spawn-ready and util` and there are 14 features and did not appear on other features doesn't that imply it is being used by other features.

Really appreciate your help.

Reply:
@tobz , thank you very much, got it, `tower/lib.rs` controls the `balance` module, and is indeed being only used there, having only `#[cfg(feature = "balance")]` does the job.